lib/llm/providers/openai.rb

```diff
@@ -45,9 +52,9 @@ def completion_model(raw)
     end
 
     ##
-    # @param (see LLM::Provider#completion_messages)
-    # @return (see LLM::Provider#completion_messages)
-    def completion_messages(raw)
+    # @param (see LLM::Provider#completion_choices)
+    # @return (see LLM::Provider#completion_choices)
+    def completion_choices(raw)
```

Comment:
I think we should consider passing the Completion instance instead of `raw`. 
We can still access `raw` via `completion.raw`. It has the potential to be more useful later on

Reply:
@0x1eef This i'm still unsure about.
The reason i left only raw is because i'm thinking of these methods in terms of parsing methods only.
Doesn't passing the completion object to `Provider#completion_messages` for example create a circular dependency?

Reply:
Alright :)